src/AnalyzeCli.cpp

```diff
@@ -134,6 +157,8 @@ int startAnalyze(int argc, char** argv)
             // Backwards compatibility
             else if (strncmp(argv[i], "--defs=", 7) == 0)
                 definitionsPaths.push_back(std::string(argv[i] + 7));
+            else if (strncmp(argv[i], "--ignoreGlobs=", 14) == 0)
+                ignoreGlobPatterns.push_back(std::string(argv[i] + 14));
```

Comment:
Probably better off just naming this `--ignore`
```suggestion
            else if (strncmp(argv[i], "--ignore=", 9) == 0)
                ignoreGlobPatterns.push_back(std::string(argv[i] + 9));
```